src/Sql/dbo/Views/SsoConfigView.sql

```diff
@@ -1,6 +1,8 @@
 ﻿CREATE VIEW [dbo].[SsoConfigView]
 AS
 SELECT
-    *
+    SSO.*
 FROM
-    [dbo].[SsoConfig]
+    [dbo].[SsoConfig] SSO
+INNER JOIN
+    [dbo].[Organization] O ON O.[Identifier] = SSO.[Identifier]
```

Comment:
Please join on the left table's clustered index for performance purposes, in this case `SSO.[OrganizationId] = O.[Id]`

Reply:
Updated logic and moved to sproc